### DISCUSSION TITLE
Flexiports are labelled as 0 and 1 instead of 1 and 2 in Smart Message UI.

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#76] [Editor App Feature Requests] [answered]
<img width="702" alt="Screen Shot 2022-08-22 at 9 37 15 am" src="https://user-images.githubusercontent.com/59382082/185840586-38725967-d52d-4df1-87c4-fd683d9d210e.png">

**Comment by konnsim** [comment, marked as answer]
Fixed.
